cli/install/install.go

```diff
@@ -434,20 +471,42 @@ func installTt(binDir string, installCtx InstallCtx, distfiles string) error {
 		}
 
 		if !versionFound {
-			return fmt.Errorf("%s version of tt doesn't exist", ttVersion)
+			commits, err_hashes := getTTCommits(installCtx.Local, distfiles)
```

Comment:
Please add debug log, and print if we are searching for a version or a commit.

Reply:
fixed